docs/products/flink/howto/real_time_alerting_solution_cli.rst

```diff
@@ -0,0 +1,428 @@
+      {"hostname": "sleepy", "cpu": "cpu1", "usage": 85.3384018012967, "occurred_at": 1633956788484}
+      {"hostname": "sneezy", "cpu": "cpu1", "usage": 89.11518629380006, "occurred_at": 1633956781891}
+      {"hostname": "sneezy", "cpu": "cpu2", "usage": 89.69951046388306, "occurred_at": 1633956788294}
+
+
+Create a pipeline for basic filtering
+-------------------------------------
+
+This setup uses a fixed threshold to filter any instances of high CPU load to a separate Kafka topic.
+
+.. mermaid::
+
+    graph LR;
+
+        id1(Kafka source)-- metrics stream -->id2(Flink job);
+        id2-- high CPU -->id3(Kafka sink);
+
+
+1. Using the Aiven CLI, run the following command to create a Kafka table named ``CPU_IN``:
+
+   ::
+
+      avn service flink table create demo-flink KAFKA_INTEGRATION_ID \
+          --table-name CPU_IN                                        \
+          --kafka-topic cpu_load_stats_real                          \
+          --schema-sql "TABLE_SQL"
+
+   Replace ``KAFKA_INTEGRATION_ID`` with the ID for your ``demo-kafka`` service integration and replace ``TABLE_SQL`` with the following:
+
+   .. literalinclude:: /code/products/flink/alerting_solution_sql.md
+      :lines: 2-8
+      :language: sql
+
+#. Run the following command to create the output table named ``CPU_OUT_FILTER``:
+
+   ::
+
+      avn service flink table create demo-flink KAFKA_INTEGRATION_ID \
+          --table-name CPU_OUT_FILTER                                \
+          --kafka-topic cpu_load_stats_real_filter                   \
+          --schema-sql "TABLE_SQL"
+
+   Replace ``TABLE_SQL`` with the following:
+
+   .. literalinclude:: /code/products/flink/alerting_solution_sql.md
+      :lines: 11-14
+      :language: sql
+
+#. Run the following command to list the tables for the ``demo-flink`` service:
+
+   ::
+
+      avn service flink table list demo-flink
+
+   The output for this command shows you the table IDs, which you need in the command that you use to create Flink jobs:
+
+   ::
+
+     INTEGRATION_ID                        TABLE_ID                              TABLE_NAME
+     ====================================  ====================================  ==========
+     917bbec0-0f34-4a31-b910-c585feb95d09  305c44d9-22d5-4be8-987f-57c7642e8a89  CPU_IN
+     917bbec0-0f34-4a31-b910-c585feb95d09  3d33a7c5-3716-4b21-9739-f79228f9f28f  CPU_OUT_FILTER
+
+#. Run the following command to create a data pipeline job named ``simple_filter``:
+
+   ::
+
+      avn service flink job create demo-flink simple_filter     \
+          --table-ids CPU_IN_ID CPU_OUT_FILTER_ID               \
+          --statement "JOB_SQL"
+
+   Replace the values for ``--table-ids`` with the IDs for the ``CPU_IN`` and ``CPU_OUT_FILTER`` tables, and ``JOB_SQL`` with the following:
+
+   .. literalinclude:: /code/products/flink/alerting_solution_sql.md
+      :lines: 17-24
```

Comment:
Please don't use line numbers for includes, it's quite difficult to maintain. Either separate files or named labels in the file would work better I think.

Reply:
Fixed.